docs/micro-rdk/motor/gpio.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Configure a gpio motor"
+linkTitle: "gpio"
+weight: 10
+type: "docs"
+description: "Configure brushed or brushless DC motors."
+images: ["/icons/components/motor.svg"]
+# SMEs: Rand, James
+---
+
+The `gpio` model supports [DC motors](https://en.wikipedia.org/wiki/DC_motor) (both brushed and brushless).
+
+You can configure [encoders](/micro-rdk/encoder/) to work with `gpio` motors.
+Find more information in the [encoded motor documentation](/components/motor/gpio/encoded-motor/).
+
+To configure a DC motor as a component of your robot, first configure the [board](/components/board/) to which the motor driver is wired.
+Then add your motor:
+
+{{< tabs name="gpio-config">}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `motor` type, then select the `gpio` model.
+Enter a name for your motor and click **Create**.
+
+![G P I O motor config in the builder UI with the In1 and In2 pins configured and the PWM pin field left blank.](/components/motor/gpio-config-ui.png)
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json
+{
+  "components": [
+    {
+      "name": "<your-board-name>",
+      "model": "<your-board-model>",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+    },
+    {
+      "name": "<your-motor-name>",
+      "model": "gpio",
+      "type": "motor",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": {
+          <...>
+        },
+        "board": "<your-board-name>",
+        "min_power_pct": <float>,
+        "max_power_pct": <float>,
+        "pwm_freq": <float>,
+        "dir_flip": <float>,
+        "en_low": <float>
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+An example configuration for a `gpio` motor:
+
+```json
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "pi",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": []
+    },
+    {
+      "name": "example-gpio",
+      "model": "gpio",
+      "type": "motor",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": {
+          "dir": "36",
+          "pwm": "32"
+        },
+        "board": "local"
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `gpio` motors:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `board` | string | **Required** | `name` of the [board](/components/board/) to which the motor driver is wired. |
```

Comment:
```suggestion
| `board` | string | **Required** | `name` of the [board](/micro-rdk/board/) to which the motor driver is wired. |
```
How about this? So that they stay within micro-RDK land?